composer.json

```diff
@@ -157,7 +157,8 @@
         "twig/inky-extra": "^2.12|^3",
         "twig/markdown-extra": "^2.12|^3",
         "web-token/jwt-checker": "^3.1",
-        "web-token/jwt-signature-algorithm-ecdsa": "^3.1"
+        "web-token/jwt-signature-algorithm-ecdsa": "^3.1",
+        "web-token/jwt-signature-algorithm-rsa": "^3.1"
```

Comment:
Is there a reason why adding support for RSA is in this PR? If it's independent, this should be submitted as a separate PR to me.

Reply:
Yes it makes sense.  
PR for RSA support is opened.
https://github.com/symfony/symfony/pull/53682

Reply:
Hi,

Please not that RSA1_5 algorithms are now deprecated. See [RFC 8725](https://datatracker.ietf.org/doc/html/rfc8725#section-3.2) for more information.

The package `web-token/jwt-signature-algorithm-rsa` contains both RSA1_5 (`RSxxx`) and RSA-OAEP (`PSxxx`) algorithms.
To me, even if the RS256 is very common, it should be avoiding when possible. I don't know what algorithms are used by Amazon Cognito, but I am pretty sure they provide alternatives to this algorithm.